src/frameName.cpp

```diff
@@ -135,13 +139,20 @@ char* FrameName::truncate(char* name, int max_length) {
 
 const char* FrameName::decodeNativeSymbol(const char* name) {
     const char* lib_name = (_style & STYLE_LIB_NAMES) ? Profiler::instance()->getLibraryName(name) : NULL;
+    bool report_lib_name_from_symbol = false;
+    if (lib_name == NULL && !endsWith(name, "_[k]", 4)) {
```

Comment:
This handles the case when `name` is a library name, then?

Reply:
i.e the case where in any case we want to show `(DSO)` because there is no symbol

Reply:
Yes, that's right.